### ISSUE TITLE
Is the License fine?

### ISSUE THREAD

**Issue by wendilinplay** [LC1332/Chinese-alpaca-lora#17]
Hey, I saw your license of luotuo-lora-7b-0.1 is apache2. and to my best knowledge, your model was finetuned on top of llama7b which is **non-commercial**,  also your code base of alpaca-lora is also only for **research**.  Are you sure you can make the license Apache2?  Let me know, cause I also want to finetune llama for commercial purpose.

**Comment by LC1332** [comment]
We may refine the license later once the luotuo project finished a certain milestone. Noticing our project didn't incuding the code of origin alpaca, I thought the code for our notebook and json file can be free to use in your commercial. However, once you clone the code from alpaca, you should not obey their license.

**Comment by wendilinplay** [comment]
What about the pertained model? llama is also non-permissive?

**Comment by LC1332** [comment]
Our LoRA model I think it's free to be used at any proj. However the base model you should check their original license

**Comment by wendilinplay** [comment]
Sorry, let me rephrase my question. Your model was fine-tuned on the original model, I thought the fine-tuned model must inherit the license from the original model. Apparently, you hasn't inherit the original one. Could you please let me know if it is allowed by Meta or maybe you just simply didn't pay attention to it?

**Comment by LC1332** [comment]
I may figure the correct license later, but the LoRA model is actually apart stored from the original one, and they must be used together. So if you want to use the model, natually you will inherit the original license.
